SPECS/python-pssh/python-pssh.spec

```
@@ -0,0 +1,81 @@
+%{?python_enable_dependency_generator}
+%global pypi_name pssh
+%global sum PSSH provides parallel version of OpenSSH and related tools
+%global desc PSSH provides parallel version of OpenSSH and related tools,\
+including pssh, pscp, prsync, pnuke and pslurp.\
+This project includes psshlib which can be used within custm applications.
+
+Name:           python-%{pypi_name}
+Version:        2.3.5
+Release:        1%{?dist}
+Summary:        %{sum}
+
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+URL:            https://github.com/lilydjwg/%{pypi_name}
+Source0:        https://github.com/lilydjwg/pssh/archive/refs/tags/v%{version}.tar.gz
```

Please use a full name for the tarball. This helps avoiding collisions and is clearer for the reader in general.
```suggestion
Source0:        https://github.com/lilydjwg/pssh/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

---

This will also require uploading the tarball with the updated name and updating the signatures/hashes file.

---

changes have been committed according to the feedback.